arelle/logging/handlers/LogHandlerWithXml.py

```diff
@@ -38,9 +43,14 @@ def propElts(properties: list[tuple[Any, Any, Any]], indent: str, truncateAt: in
                                if 2 <= len(p) <= 3)
 
         msg = self.format(logRec)
+        if self.logXmlMaxAttributeLength is not None:
+            xmlTruncation = self.logXmlMaxAttributeLength
+        else:
+            xmlTruncation = DEFAULT_LOG_XML_MAX_LENGTH
         if logRec.args and isinstance(logRec.args, Mapping):
-            args = "".join([' {0}="{1}"'.format(ncNameEncode(n), entityEncode(v,
-                                                                              truncateAt=(4096000 if n in ("json",) else 128)))
+            args = "".join([' {0}="{1}"'.format(
+                ncNameEncode(n), entityEncode(v, truncateAt=(JSON_MAX_LENGTH if n in ("json",) else (xmlTruncation)))
```

Comment:
The parentheses around xmlTruncation are unnecessary and preexisting, but the in tuple check is also unnecessary.
```suggestion
            args = "".join([' {0}="{1}"'.format(
                ncNameEncode(n), entityEncode(v, truncateAt=(self.maxJsonAttributeLength if n == "json" else self.maxXmlAttributeLength))
```